output/analysis/plot.eps

```diff
@@ -385,18 +385,47 @@
 } def
 /Helvetica /Helvetica-0 MF-Helvetica
 /Slw 0.120 def
-1.000 1.000 1.000 setrgbcolor
+0.882 0.902 0.941 setrgbcolor
```

Comment:
I think you need to change the default back to s1mono. The color in this eps is not standard.